swagger_server/itm/data/dryrun/test/sample.yaml

```diff
@@ -186,13 +186,13 @@ scenes:
     end_scene_allowed: false
     next_scene: evac_decision
     probe_config:
-      - probe_id: adept-september-demo-probe-1
```

Comment:
Not sure if this is necessary, however the other `sample.yaml` located in `itm/data/metrics/test` still has `adept-september-demo-probe-X`

Reply:
Yeah, my thinking (hope?) is that we're not going to continue to update metrics scenarios as the API evolves, including the sample files.  That said, this is a good thing to check with @brianpippin .  Do we need these to stay in a runnable state?

Reply:
I went ahead and updated the metrics samples.  Easy to do, and no harm in it.